test/test_grpc.py

```diff
@@ -46,86 +46,50 @@ class TestCreateStub(unittest2.TestCase):
+        secure_authorized_channel.assert_called_once_with(
+            credentials,
             '{}:{}'.format(self.FAKE_SERVICE_PATH, self.FAKE_PORT),
-            comp.return_value)
-        auth.assert_called_once_with(fake_scopes)
-
-    @mock.patch('grpc.metadata_call_credentials')
-    @mock.patch('grpc.composite_channel_credentials')
-    @mock.patch('grpc.ssl_channel_credentials')
-    @mock.patch('grpc.secure_channel')
-    @mock.patch('google.gax.auth.make_auth_func')
+            ssl_credentials=None)
+
+        self.assertEqual(got_channel, secure_authorized_channel.return_value)
+
+    @mock.patch('google.gax.auth.get_default_credentials')
+    @mock.patch('google.gax.auth.secure_authorized_channel')
     def test_creates_a_stub_with_given_channel(
-            self, auth, chan, chan_creds, comp, md):
-        fake_channel = object()
+            self, secure_authorized_channel, get_default_credentials):
+        fake_channel = mock.Mock()
         got_channel = grpc.create_stub(
-            _fake_create_stub, self.FAKE_SERVICE_PATH, self.FAKE_PORT,
-            channel=fake_channel)
+            _fake_create_stub, channel=fake_channel)
         self.assertEqual(got_channel, fake_channel)
-        self.assertFalse(auth.called)
-        self.assertFalse(chan_creds.called)
-        self.assertFalse(chan.called)
-        self.assertFalse(comp.called)
-        self.assertFalse(md.called)
-
-    @mock.patch('grpc.metadata_call_credentials')
-    @mock.patch('grpc.composite_channel_credentials')
-    @mock.patch('grpc.ssl_channel_credentials')
-    @mock.patch('grpc.secure_channel')
-    @mock.patch('google.gax.auth.make_auth_func')
-    def test_creates_a_stub_ok_with_given_creds(self, auth, chan, chan_creds,
-                                                comp, md):
-        fake_creds = object()
-        got_channel = grpc.create_stub(
-            _fake_create_stub, self.FAKE_SERVICE_PATH, self.FAKE_PORT,
-            ssl_creds=fake_creds)
-        chan.assert_called_once_with(
-            '{}:{}'.format(self.FAKE_SERVICE_PATH, self.FAKE_PORT),
-            comp.return_value)
-        auth.assert_called_once_with([])
-        self.assertTrue(chan.called)
-        self.assertFalse(chan_creds.called)
-        self.assertTrue(comp.called)
-        self.assertTrue(md.called)
-        self.assertEqual(got_channel, chan.return_value)
-
-    @mock.patch('grpc.composite_channel_credentials')
-    @mock.patch('grpc.ssl_channel_credentials')
-    @mock.patch('grpc.secure_channel')
-    @mock.patch('google.gax.auth.make_auth_func')
-    def test_creates_a_stub_ok_with_given_auth_func(self, auth, dummy_chan,
-                                                    dummy_chan_creds, dummy_md):
-        grpc.create_stub(
-            _fake_create_stub, self.FAKE_SERVICE_PATH, self.FAKE_PORT,
-            metadata_transformer=lambda x: tuple())
-        self.assertFalse(auth.called)
+        self.assertFalse(secure_authorized_channel.called)
```

Comment:
You should pass these variables as parameters to `grpc.create_stub`; otherwise this doesn't actually test anything.